### ISSUE TITLE
Usage example needs [CLS] and [SEP] added post-tokenization

### ISSUE THREAD

**Issue by jplehmann** [huggingface/pytorch-pretrained-BERT#342]
Since probably #176, the usage example results in the special tokens getting normalized in a bad way and the assertion clearly fails.
```
['[',
 'cl',
 '##s',
 ']',
 'who',
 'was',
 'jim',
 'henson',
 '[MASK]',
 '[',
 'sep',
 ']',
 'jim',
 'henson',
 'was',
 'a',
 'puppet',
 '##eer',
 '[',
 'sep',
 ']']
```
I believe something like this is intended:
```
text1 = "Who was Jim Henson ?"
text2 = "Jim Henson was a puppeteer"
tokenized_text = ['[CLS]'] + tokenizer.tokenize(text1) + ['[SEP]'] + tokenizer.tokenize(text2) + ['[SEP]']
```

I really appreciate this project!

**Comment by jplehmann** [comment]
Argh, I just realized that due to dependency conflicts, pip had installed an old version `0.3.0`.

Was fixed here:
https://github.com/huggingface/pytorch-pretrained-BERT/issues/303
